boards/nxp/frdm_mcxn947/board.c

```diff
@@ -132,6 +132,14 @@ static int frdm_mcxn947_init(void)
 	CLOCK_EnableClock(kCLOCK_Gpio5);
 #endif
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(wwdt0), okay)
+	CLOCK_SetClkDiv(kCLOCK_DivWdt0Clk, 1u);
+#endif
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(wwdt1), okay)
+	CLOCK_SetClkDiv(kCLOCK_DivWdt1Clk, 1u);
+#endif
```

Comment:
these changes should not be in the driver commit

Reply:
Need to be tied to the device tree changes I'm guessing? Please confirm @decsny if this is what you are suggesting here with this comment.

Reply:
The board change files should not be clubbed with the driver commit. It should go in the commit with other board changes.

Reply:
No longer applicable.